myosuite/tests/test_myochallenge_track.py

```diff
@@ -0,0 +1,33 @@
+import gymnasium 
+import myosuite
+from mujoco import viewer
+import time
+
+
+def run_env():
```

Comment:
Check how others envs are being tested and integrate with it

Reply:
This was a duplicate test. Removed it